jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/TypeRule.java

```diff
@@ -78,8 +80,8 @@ public JType apply(String nodeName, JsonNode node, JsonNode parent, JClassContai
     if (propertyTypeName.equals("object") || node.has("properties") && node.path("properties").size() > 0) {
 
       type = ruleFactory.getObjectRule().apply(nodeName, node, parent, jClassContainer.getPackage(), schema);
-    } else if (node.has("existingJavaType")) {
-      String typeName = node.path("existingJavaType").asText();
+    } else if (hasExtensionProperty(node, "existingJavaType")) {
```

Comment:
while you are in here, does it make sense to move all these string ("existingJavaType" etc.) to a constant and just refer to it?

Reply:
Well.... I don't know.

I get what you mean, and that is my instinct as well. But the code author didn't use them and I'm not looking to clean up his code, per se. Not changing them has one benefit of an easier code review because the review highlights show that the fixed-string never changes (thus no obvious regressions). 

Since I don't know why someone doesn't use constants in the first place I don't know how they would feel about introducing them here, but not everywhere else in the code where the various standard spec keywords are discovered and operated on.   And tracking down all other fixed-string uses seems like it would just muddy an already large pull request.

I don't mind doing it, but I don't know if it makes sense. Thoughts?

Reply:
I like your point that it is nice to be able to see that there is not a regression.  As a follow up someone could find the method call and replace with constants.